microsoftgraph/microsoft-graph-explorer
# Fixes ExpressionChangedAfterItHasBeenChecked

## Overview

Fixes ExpressionChangedAfterItHasBeenChecked

### Demo

![console](https://user-images.githubusercontent.com/12011447/47629585-f1787180-db4c-11e8-9318-cc26b190ca9a.JPG)

## Testing Instructions

* Launch Graph Explorer
* Open the developer tools
* Observe that the error `ExpressionChangedAfterItHasBeenChecked` is not thrown
